docs/app/insights.md

```diff
@@ -1,81 +1,199 @@
+unselectable in the field dropdown list. :::
+
+#### Style and Format Options
+
+- **Abbreviate Value** - Abbreviates large numbers with a letter _(e.g. 2,000 = 2k)_
+- **Decimals** - Changes the number of decimals displayed
+- **Prefix** - Adds text before the metric
+- **Suffix** - Adds text after the metric
+- **Conditional Styles** - Changes color of number displayed if it is `>`, `>=`, `<`, `<=`, `==` (equal to), or `!=`
+  (not equal to) to some defined value
+
+### Time Series Overview
+
+The Time Series panel allows you to build graphs and see how data changes over time.
+
+::: tip Note In order to use this panel, your collection will need a datetime field. :::
+
+![image of Time Series panel options](image.jpg)
+
+- **Collection** - Selects the collection you want to analyze
+- **Color** - Sets color of your graph curve
+- **Group Aggregation** - Selects the type of aggregation to perform
+- **Group Precision** - Sets the time interval to aggregate by _(days, weeks, months etc.)_
+- **Date Field** - Selects the datetime field used along the x-axis.
+- **Date Range** - Selects the time range from which data should be displayed
+- **Value Field** - Sets the field you wish to aggregate
+- **Value Decimals** - Changes the number of decimals displayed on the y-axis
+- **Min Value** - Sets minimum value displayed on y-axis
+- **Max Value** - Sets maximum value displayed on y-axis
+- **Curve Type** - Sets curve to be shown as smooth, straight or stepline
+- **Fill Type** - Sets fill type under curve to gradient, solid, or none
+- **Filter** - Allows filtering of items considered for analysis based on logical conditions
+- **Show X-axis** - Toggles display of time along X-axis
+- **Show Y-axis** - Toggles display of numeric values along Y-axis
+
+### Advanced Filtering
+
+The filtering options are infinitely customizable. However, they will be limited by the collections, relations, and
+information stored in your database.
+
+#### Nested Filters
+
+Sometimes, you will need to run filters that require information from two or more collections. Here might be a few
+examples:
+
+- Highest paid authors that wrote 5 or more books (uses `authors` and `books` collections)
+- Products with less than 50 sales monthly (uses `products` and `orders` collections)
+- Blog Posts with the highest number of social shares (uses `blog` and `social share metadata` collections )
+
+Nested filters limit results of one collection by the values of fields from a related collection. Let's look at how to
+create a nested filter:
+
+![Image of a nested filter with numbers](image.jpg)
+
+1. Create a filter
+2. Select a foreign key field
+3. Create a new filter
+4. Nest it under the first filter by clicking and dragging the lower filter to the right
+5. Select the filters to apply on this collection
+
+::: tip Foreign Key A _foreign key field_ is a field that references the primary key (often times this is the `ID`)
```

Comment:
Let's reference global explanations... this should link to the /Glossary page

Reply:
Foreign Key is not in the glossary yet.
(But I can add it real soon)

Reply:
"Mirror" it?? You mean add it into the glossary section? @benhaynes

Reply:
Yup... copy/paste the PK to create a FK section

Reply:
> > [[toc]]
> > Doesn't add in ## and ### headers. Couldn't find this notation after multiple rounds of googling. 
>
> That sounds like a VuePress configuration to me. There might be configuration flag somewhere to have it inject h2 (`##`) and h3 (`###`) headers.
> 
Okay, I haven't messed with VuePress ever, but will take some time to get to know it this week.

> > Should the definition of Aggregate Functions go into the Glossary or the insights doc? (I think it should go in the glossary)
> 
> I think in the glossary.
> 
Cool cool!

> > Hyperlinks to other docs don't jump to /#section-header when testing.
> 
> Odd, how are you testing that exactly? Does the same problem exist on the live docs website?
> 
I simply just opened the docs in dev mode and tried to click on the hyperlinks.

> > This will eventually become a full page. for now, the definition is:    Defines how list items will be represented. Allows use of field item values as well as custom text. ... Later on it will be hyperlinked. Will that be good enough for the time being? Or should I expand this asap?
> 
> I think it's fine for now. Having it is better than not having it, and we can't get too stuck on making it perfect in the first cycle. We'll have to really treat this docs upgrade as an iterative process, otherwise we'll spend an infinite amount of fine-tuning, and never go live
> 

Sweet. I'm all about it. These last few days my understanding of the scope of the docs has expanded quite a bit!

> > Video delivered in iframe... Not sure how we want to handle the HTML/CSS for it.
> 
> Should be as simple as
> 
> ```
> <video autoplay muted loop>
>   <source src="https://cdn.directus.io/etc/etc/etc.mp4" type="video/mp4" />
> </video>
> ```

Using `<iframe>` made it a tiny little icon with black padding on the sides. `<Video>` delivers it full sized, without rounded edges. Some custom CSS will need to be added. It doesn't fit to column and. (I can do that but I have taken a peek at Dircetus CSS and it will take me a while to be confident I'm matching your CSS styling nuances properly)

> > When I run git commit, it reformats the ::: tip and ::: warning syntax causing a UI bug! Perhaps it has to do with a plugin in VSC... but then why would it only reformat on commits?
> 
> This is most likely a formatting issue then. Make sure to always have a whiteline before and after the `:::` block for it to work properly:
> 
> ```
> Some other text
> 
> ::: tip Example!
> 
> This is the content of the tip block
> 
> :::
> 
> Some other paragraph
> ```

There is a blank line before and after. I will go ham on google though to try and sort this out though.